src/sentry/api/urls.py

```diff
@@ -2334,6 +2334,11 @@
         SharedGroupDetailsEndpoint.as_view(),
         name="sentry-api-0-shared-group-details",
     ),
+    url(
+        r"^shared/(?P<organization_slug>[^\/]+)/(?:issues|groups)/(?P<share_id>[^\/]+)/$",
```

Comment:
I'm a little concerned about placing the organization slug at various places inside the URL structure, rather than following the "REST-ful noun" pattern of `organizations/(?P<organization_slug>[^\/]+)` at the beginning. (I'm not sure how many other mappings are already doing this -- I noticed at least one, after I'd already hit "approve" on it the other day.)

In general, it seems like it's inviting ambiguity. In this case, it looks like we'd be relying on not having an org with "issues" or "groups" as its slug (I don't know if those words are already included in reserved slugs).

When adding org slugs to mappings, would it be plausible to just always follow the pattern of "`organizations/` + org slug + old mapping"? Or am I assuming too much?

Reply:
it should definitely be possible... i can follow up this PR with another one to modify the ones i've already changed

Reply:
my original train of thought for this was to put the org slug after the noun describing what the url does

Reply:
see #42825